eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/http/processor/inf/HttpRequestProcessor.java

```diff
@@ -67,4 +68,10 @@ default String getExtension(CloudEvent event, String protocolKey) {
         return Objects.isNull(extension) ? "" : extension.toString();
     }
 
+    /**
+     *
+     * @return {@link Executor}
+     */
+    Executor executor();
+
```

Comment:
It seems this method has no functional relation with `HttpRequestProcessor` interface. As for the current implementation of EventMesh for asynchronous requests, the executor has not been placed within the interface that handles the requests. May you please explain its design?